tgui/packages/tgui/store.ts

```diff
@@ -56,12 +56,10 @@ export const configureStore = (options: ConfigureStoreOptions = {}): Store => {
         ...(middleware?.post || []),
       ];
 
-  if (process.env.NODE_ENV !== 'production') {
+  if (process.env.NODE_ENV !== 'production' && sideEffects) {
     // We are using two if statements because Webpack is capable of
     // removing this specific block as dead code.
```

Comment:
this comment is no longer relevant if you're making it one if statement now

Reply:
This comment is actually about why you should not join these if statements. It's a webpack quirk. @jlsnow301